tools/kotlin-native-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/tasks/KonanCompileTask.kt

```diff
@@ -106,21 +125,21 @@ abstract class KonanCompileTask: KonanBuildingTask(), KonanCompileSpec {
         addKey("-ea", enableAssertions)
         addKey("--time", measureTime)
         addKey("-nodefaultlibs", noDefaultLibs)
+        addKey("-Xmulti-platform", commonProject != null)
```

Comment:
Would it be a problem if 'Xmulti-platform' would be passed explicitly by user in gradle script?

Reply:
Looks like it wouldn't but I'll double-check it.